.gitignore

```diff
@@ -90,6 +90,9 @@ public
 tmp/
 temp/
 
+# PR description files
+pr-description.md
```

Comment:
This seems a silly, abritrary filename to exclude.  Contributors should just not commit irrelevant files.